pkg/controllers/provisioningv2/rke2/provisioningcluster/template.go

```diff
@@ -144,7 +145,8 @@ func toMachineTemplate(machinePoolName string, cluster *rancherv1.Cluster, machi
 	}
 
 	if machinePool.DynamicSchemaSpec == "" {
-		return nil, fmt.Errorf("waiting for dynamic schema to be populated for machine pool %s", machinePoolName)
+		logrus.Debugf("rkecluster %s/%s: waiting for dynamic schema to be populated for machine pool %s", cluster.Namespace, cluster.Name, machinePoolName)
```

Comment:
Nit, but can we keep the message format consistent with another log message we produce in this file?
https://github.com/rancher/rancher/blob/cd1a71624a94de993d471ce8a643cf4d2dbb75a6/pkg/controllers/provisioningv2/rke2/provisioningcluster/template.go#L485

Reply:
I was using this format: https://github.com/rancher/rancher/blob/cd1a71624a94de993d471ce8a643cf4d2dbb75a6/pkg/controllers/provisioningv2/rke2/provisioningcluster/controller.go#L309

Might be a good idea to settle on a log format and update all the logs mesages in this file, @Oats87 thoughts? Thinking `[provcluster] %s/%s:` since `rkecluster` is the infrastructure cluster, which is another type entirely.

Reply:
also, having `rkecluster` in every message actually gives a `grep` keyword that correlates to the `v2prov` controller messages